runtime/doc/api.txt

```diff
@@ -1307,6 +1307,32 @@ nvim_put({lines}, {type}, {after}, {follow})                      *nvim_put()*
                   |P|).
       • {follow}  If true place cursor at end of inserted text.
 
+nvim_redraw({opts})                                            *nvim_redraw()*
+    Instruct Nvim to redraw various elements.
+
+    Parameters: ~
+      • {opts}  Optional parameters.
+                • `update_screen`: Immediately update the screen. Useful when
```

Comment:
```suggestion
                • `screen`: Immediately update the screen. Useful when
```
update (=redraw) is implied

Reply:
In general, we should try to be consistent about "update" vs. "redraw" here (or, if there is a distinction, explain it).

Reply:
The implication of `update_screen` is that the actual buffer content in windows is redrawn, for which `update_screen()` is the only entry point internally. As opposed the various other "widgets" associated with a window which can also be targeted in this API and redrawn without redrawing the entire window, but would also be redrawn when `update_screen()` is called and they were pending for redraw.

Considering that, maybe this can be called `text` or `viewport` as it is referred to under `:h window`?

Reply:
Thanks for the explanation. So `flush` is (to me!) closest in meaning (but still needs an explanation at this point in the documentation, or at least a taglink), but I defer to @bfredl